nbextensions/bulkImportCell/tabs/configure.js

```diff
@@ -158,21 +86,58 @@ define([
         }
 
         function buildFilePathWidget(node) {
-            const paramBus = runtime
-                .bus()
-                .makeChannelBus({ description: 'Parent comm bus for input widget' });
             // This is the key in the model that maps to the list of params for the current app.
-            const paramKey = `params.${fileType}`;
+            const paramBus = buildMessageBus(fileType, 'Parent comm bus for filePath widget');
+            paramBus.on('parameter-changed', (message) => {
+                updateModelParameterValue(fileType, FILE_PATH_TYPE, message);
+            });
+            paramBus.on('sync-data-model', (message) => {
```

Comment:
This gets called by the filePathWidget when it makes a change like removing a row.